packages/cactus-plugin-bungee-hermes/src/main/typescript/view-creation/view.ts

```diff
@@ -2,12 +2,29 @@ import { v4 as uuidV4 } from "uuid";
 import { Snapshot } from "./snapshot";
 import MerkleTree from "merkletreejs";
 import { Transaction } from "./transaction";
+import { ViewPrivacyPolicy } from "./privacy-policies";
+import * as CryptoJS from "crypto-js";
+
+export interface ViewMetadata {
+  viewId: string;
+  viewProof: {
+    transactionsMerkleRoot: string;
+    statesMerkleRoot: string;
+  };
+  policy?: { policyId: string; policyHash: string };
+  creator: string;
+  signature: string;
+}
+
 export class View {
   private key: string;
   private snapshot: Snapshot;
   private tI: string;
   private tF: string;
   private participant: string;
+  private creator: string = "";
```

Comment:
this should be undefined at the beginning (we should not allow an empty creator)